docs/user-guide/jobs.md

```diff
@@ -373,7 +373,3 @@ driver, and then cleans up.
 
 An advantage of this approach is that the overall process gets the completion guarantee of a Job
 object, but complete control over what pods are created and how work is assigned to them.
-
-## Future work
-
-Support for creating Jobs at specified times/dates (i.e. cron) is expected in [1.4](https://github.com/kubernetes/kubernetes/pull/11980).
```

Comment:
Instead of having a link at the bottom I went with adding SJ link on the left side, in the menu, which I think will make more sense.